frontend/src/component/events/EventTimeline/EventTimelineEvent/EventTimelineEventTooltip/EventTimelineEventTooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import type { EventSchema } from 'openapi';
+import { formatDateYMDHMS } from 'utils/formatDate';
+
+interface IEventTimelineEventTooltipProps {
+    event: EventSchema;
+}
+
+export const EventTimelineEventTooltip = ({
+    event,
+}: IEventTimelineEventTooltipProps) => {
+    const { locationSettings } = useLocationSettings();
+    const eventDateTime = formatDateYMDHMS(
+        event.createdAt,
+        locationSettings?.locale,
+    );
+
+    if (event.type === 'feature-environment-enabled') {
+        return (
+            <div>
+                <small>{eventDateTime}</small>
+                <p>
+                    {event.createdBy} enabled {event.featureName} for the{' '}
+                    {event.environment} environment in project {event.project}
+                </p>
+            </div>
+        );
+    }
+    if (event.type === 'feature-environment-disabled') {
+        return (
+            <div>
+                <small>{eventDateTime}</small>
+                <p>
+                    {event.createdBy} disabled {event.featureName} for the{' '}
+                    {event.environment} environment in project {event.project}
+                </p>
+            </div>
+        );
+    }
+
+    return (
+        <div>
+            <div>{eventDateTime}</div>
+            <div>{event.createdBy}</div>
+            <div>{event.type}</div>
+            <div>{event.featureName}</div>
+            <div>{event.environment}</div>
+        </div>
+    );
+};
```

Comment:
This needs to be properly implemented. Will be addressed at a later point.

Reply:
Cool, this looks like a rendering strategy. Agree to do it later!